ci-operator/step-registry/optional-operators/cvp-common/scorecard/optional-operators-cvp-common-scorecard-commands.sh

```diff
@@ -102,10 +102,6 @@ if [ -f "${OPERATOR_DIR}/tests/scorecard/config.yaml" ]; then
 
   # Use the json output format for cvp custom scorecard tests
   CUSTOM_SCORECARD_OUTPUT_FORMAT="json"
-  # Use the xunit output format for msp custom scorecard tests
-  if [ "${TEST_MODE}" == "msp" ]; then
-    CUSTOM_SCORECARD_OUTPUT_FORMAT="xunit"
-  fi
```

Comment:
Could we make it configurable ? Or will be done it in other story?
Cause `TEST_MODE` and `OUTPUT_FORMAT` should not be tight at all.

Reply:
I agree we should make this configurable. But let's open a new story.